cvFit/src/app/form/auth.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from "@angular/core";
+import { IUser } from './user.model';
+
+@Injectable()
+export class AuthService {
+    currentUser: IUser | undefined ;
+    loginUser(userName: string, password: string){
+        this.currentUser = {
+            id: 1,
+            userName: userName,
+            firstName: 'Tommy',
+            lastName: 'Johnson'
+        }
+    }
+    isAuth(){
```

Comment:
Make sure you're using Typescript to the fullest. If your function returns a type you need to declare it. _Most_ of the time an auth call is going to return an observable of a boolean because it probably has to look up to see if tokens are valid and things like that. Here's an example:

```typescript
public isAuthenticated(): Observable<boolean> { ...  }
```

Reply:
Same wiht Login user, it will more than likely be an observable. You can fake observables by using `of(...)`

Reply:
No change here for now though. Let's get this initial commit it